templates/docs/rhdm74-authoring-ha.adoc

```diff
@@ -59,7 +59,7 @@ https://docs.openshift.org/latest/architecture/core_concepts/templates.html#para
 |`ES_HTTP_PORT` | `ES_HTTP_PORT` | Sets the ES http.port property. This will set the http port of the main ES cluster node. Used to interact with cluster rest api. | `${ES_HTTP_PORT}` | False
 |`ES_JAVA_OPTS` | `ES_JAVA_OPTS` | Appends custom jvm configurations/properties to ES jvm.options configuration file. | `${ES_JAVA_OPTS}` | False
 |`AMQ_IMAGE_STREAM_NAMESPACE` | -- | Namespace in which the ImageStream for the AMQ image is installed. Default is "openshift". | openshift | True
-|`AMQ_IMAGE_STREAM_NAME` | -- | The name of the image stream to use for the AMQ broker. Default is "amq-broker72-openshift". | amq-broker72-openshift | True
+|`AMQ_IMAGE_STREAM_NAME` | -- | The name of the image stream to use for the AMQ broker. Default is "amq-broker-73-openshift". | amq-broker-73-openshift | True
```

Comment:
Just an hit, you don't need to update the adoc files manually, it is auto generated before the release tag is created.

Reply:
@spolti ack, thanks!